pylint/lint/pylinter.py

```diff
@@ -1212,14 +1212,17 @@ def _astroid_module_checker(self):
         for checker in reversed(_checkers):
             checker.close()
 
-    def get_ast(self, filepath, modname, data=None):
+    def get_ast(
+        self, filepath: str, modname: str, data: Optional[str] = None
+    ) -> nodes.Module:
         """Return an ast(roid) representation of a module or a string.
 
         :param str filepath: path to checked file.
         :param str modname: The name of the module to be checked.
         :param str data: optional contents of the checked file.
         :returns: the AST
         :rtype: astroid.nodes.Module
```

Comment:
```suggestion
```
isn't that generated from the typing ?

Reply:
Partially, but the additional text isn't. No reason to remove it, somebody took the effort to write it sometime.